### PR TITLE
Move files to generate windows installer under dist/windows

### PR DESCRIPTION
**Pull request by ret2libc**
# WAIT: download and try to install both static-windows and windows installer before

**Closing issues**

Address the windows part of #743

### CODE REVIEW

**Review by XVilka — CHANGES_REQUESTED** [review]
Remove exe and add output in gitignore.

File: dist/windows/build_windows_installer.ps1 [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+$env:Path += ";$env:ProgramFiles\7-Zip"
+$env:Path += ";$env:ProgramFiles (x86)\Inno Setup 6"
+
+dist\windows\vsdevenv.ps1 $bits
+
+meson --buildtype=release --prefix=$installdir $builddir $meson_options
+ninja -C $builddir
+ninja -C $builddir install
+7z a dist\windows\Output\rizin-$name-$version.zip $installdir
+iscc dist\windows\rizin.iss /DRizinLocation=$installdir\* /DLicenseLocation=$PWD\COPYING.LESSER /DMyAppVersion=$version
```
Comment (XVilka) [inline dist/windows/build_windows_installer.ps1:24]: I think you will need to change also the license path

**Review by XVilka — CHANGES_REQUESTED** [review]
I wonder if it makes sense to add a couple sentences in PACKAGING or BUILDING then about the installer

File: dist/windows/build_windows_installer.ps1 (hunk 1 above)
Comment (ret2libc) [reply dist/windows/build_windows_installer.ps1:24]: I've already changed it. It was LICENSEbefore and it was a duplicated license inside the rizin-win-installer repository. I'm now using the COPYING.LESSER file that is already in the rizin repo

**Review by XVilka — APPROVED** [review]

**Comment by XVilka** [comment]
```
Run .github\vsdevenv.ps1 32
.github\vsdevenv.ps1: D:\a\_temp\4abec599-a27a-42ba-8d32-b496a2c53cf0.ps1:2
Line |
   2 |  .github\vsdevenv.ps1 32
     |  ~~~~~~~~~~~~~~~~~~~~
     | The module '.github' could not be loaded. For more information, run 'Import-Module .github'.

Error: Process completed with exit code 1.
```

**Review by XVilka — APPROVED** [review]

**Review by XVilka — APPROVED** [review]

**Pull request merged**